.github/scripts/check-pr.js

```diff
@@ -5,73 +5,274 @@ const { GITHUB_REPOSITORY, GITHUB_PR_NUMBER } = process.env;
 
```

Comment:
The changes in this file seem out of context for this PR. It looks like this commit should be cherry-picked into PR [#2990](https://github.com/hashgraph/hedera-json-rpc-relay/pull/2990) instead.

The `PR Label and Milestone Check` CI in this PR failed because the associated issue for this PR, [#3045](https://github.com/hashgraph/hedera-json-rpc-relay/issues/3045), does not have a milestone, as mentioned [here](https://github.com/hashgraph/hedera-json-rpc-relay/actions/runs/11170693790/job/31054001091?pr=3061#step:5:8). Simply updating the issue with the correct milestone and rerunning the `PR Label and Milestone Check` CI should resolve the failure.

Reply:
Correct.  Thanks for pointing this out.  It's now fixed.  I used this PR with the missing milestone to test the check-pr script, but forgot to push it to the correct branch.